executors.go

```diff
@@ -60,13 +61,22 @@ func (c *Connection) ValidateAndSave(model interface{}, excludeColumns ...string
 
 var emptyUUID = uuid.Nil.String()
 
+// IsZeroOfUnderlyingType will check if the value of anything is the equal to the Zero value of that type
+// using reflect.
```

Comment:
"using reflect" isn't necessary. It should tell about the what, not about the how.